rust/examples/test-drive-net.rs

```diff
@@ -20,16 +23,35 @@ struct Opt {
         help = "The URL of the coordinator"
     )]
     url: String,
+
     #[structopt(default_value = "4", short, help = "The length of the model")]
     len: u32,
+
     #[structopt(
         default_value = "1",
         short,
         help = "The time period at which to poll for service data, in seconds"
     )]
     period: u64,
+
     #[structopt(default_value = "10", short, help = "The number of clients")]
     nb_client: u32,
+
+    #[structopt(
+        short = "der",
```

Comment:
Do we need a setting for der certs and one for pem certs? Couldn't we just check the extension?

Reply:
good idea, i'll change that.